CI/CD template changes for GitLab/GitHub (#2619)
astronomer/docs: astro/ci-cd-templates/azure-devops.md

Use the following CI/CD templates to automate deploying Apache Airflow DAGs from a Git repository to Astro with [Azure DevOps](https://dev.azure.com/).

> The templates for Azure DevOps use the [image based deploy](template-overview.md#template-types) process with a _single branch implementation_, which requires only one Astro Deployment.

Reviewer: ```suggestion
The template for Azure DevOps is based on the [image deploy template](template-overview.md#template-types) with a _single branch implementation_, which requires only one Astro Deployment.
```
